test/test_tensor_creation_ops.py

```diff
@@ -940,9 +941,9 @@ def test_linspace(self, device, dtype):
                          atol=0, rtol=0)
 
         # Check for race condition (correctness when applied on a large tensor).
-        if dtype not in (torch.int8, torch.uint8, torch.int16, torch.half):
+        if dtype not in (torch.int8, torch.uint8, torch.int16, torch.half, torch.bfloat16):
             y = torch.linspace(0, 1000000 - 1, 1000000, device=device, dtype=dtype)
-            cond = y[:-1] < y[1:]
+            cond = torch.abs(y[:-1]) < torch.abs(y[1:])  # abs is to take care of complex
```

Comment:
I think you can check here the the linear interpolation works for the real and imaginary parts of the complex tensor

Reply:
The change should have also addressed your comment:

> complex inputs interpolate the real and imaginary parts separately and as expected